### DISCUSSION TITLE
deepin ISO如何进入 live 模式

### DISCUSSION THREAD

**Discussion by gfdgd-xi** [linuxdeepin/developer-center#7241] [Q&A | 问答板块]
按照以前 deepin 20 的方法修改 grub，如下：
![image](https://github.com/linuxdeepin/developer-center/assets/73065138/7953fd44-adfe-4adc-85b9-cadade53f2a8)
最后会提示如下内容，无法进入桌面
![image](https://github.com/linuxdeepin/developer-center/assets/73065138/e1da8616-922d-4326-bc31-d47abaf8a715)
现在 deepin23 要如何进入 live 模式？

**Comment by gfdgd-xi** [comment]
补充：镜像是 deepin 23 beta3 的

**Comment by BLumia** [comment]
deepin 的 iso 目前不提供 live 支持吧。可以考虑自己定制镜像（

**Reply by gfdgd-xi** [reply]
有尝试自己定制过，但也是和上图类似的错误，用xfce4就没有问题，我想知道是不是DDE有什么限制
